frontend/src/metabase/visualizations/components/settings/ColumnItem.styled.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from "@emotion/styled";
+import { color } from "metabase/lib/colors";
+
+import Icon from "metabase/components/Icon";
+
+export const ColumnItemRoot = styled.div`
+  margin: 0.5rem 0;
```

Comment:
Let's add a bg color here to avoid items being transparent:
<img width="316" alt="Screen Shot 2022-08-19 at 7 33 51 PM" src="https://user-images.githubusercontent.com/14301985/185654828-235b2e89-9f60-4af7-a2ad-1c1ec8fda411.png">

```background: ${color("white")};```